File: April-18/19-04-18/python/02-Chandu-and-Consecutive-letters.py
for _ in range(int(input())):
    data = input()
    n = len(data)
    nstr = ""
    i = 0
    while(i < n):
        nstr += data[i]
        while(i < n-1 and data[i] == data[i+1]):
            i += 1
        i += 1
    print(nstr)

File: April-18/19-04-18/java/02-Chandu-and-Consecutive-letters.java

import java.io.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(input.readLine());
        while(t-- > 0)
        {
            String str, nstr = "";
            str = input.readLine();
            int n = str.length();
            
            for(int i = 0; i < n; i++)
            {
                nstr += str.charAt(i);
                while(i < n-1 && str.charAt(i) == str.charAt(i+1))
                {
                    i++;
                    
                }
            }
            System.out.println(nstr);
        }
        
    }
}
